### PR TITLE
[helm] Bump helm Loki and GEL version

### PR DESCRIPTION
**Pull request by trevorwhitney**
**What this PR does / why we need it**:

Bumps Loki to 2.7.0 and GEL to 1.6.0 in the helm chart.

### CODE REVIEW

**Review by DylanGuedes — APPROVED** [review]
lgtm

File: production/helm/loki/values.yaml [hunk 1]
```diff
@@ -320,8 +320,6 @@ enterprise:
     registry: docker.io
     # -- Docker image repository
     repository: grafana/enterprise-logs
-    # -- Overrides the image tag whose default is the chart's appVersion
-    tag: v1.4.0
```
Comment (jeschkies) [inline production/helm/loki/values.yaml:324]: Why was this removed?

Comment (trevorwhitney) [reply production/helm/loki/values.yaml:324]: it's not used
